bench_test.go

```diff
@@ -0,0 +1,96 @@
+package cose_test
+
+import (
+	"io"
+	"testing"
+
+	"github.com/veraison/go-cose"
+)
+
+func newSign1Message() *cose.Sign1Message {
+	return &cose.Sign1Message{
+		Headers: cose.Headers{
+			Protected: cose.ProtectedHeader{
+				cose.HeaderLabelAlgorithm: cose.AlgorithmES256,
+			},
+			Unprotected: cose.UnprotectedHeader{
+				cose.HeaderLabelKeyID: 1,
+			},
+		},
+		Payload:   make([]byte, 100),
+		Signature: make([]byte, 32),
+	}
+}
+
+type noSigner struct{}
+
+func (noSigner) Algorithm() cose.Algorithm {
+	return cose.AlgorithmES256
+}
+
+func (noSigner) Sign(_ io.Reader, digest []byte) ([]byte, error) {
+	return digest, nil
+}
+
+type noVerifier struct{}
+
+func (noVerifier) Algorithm() cose.Algorithm {
+	return cose.AlgorithmES256
+}
+
+func (noVerifier) Verify(_, _ []byte) error {
+	return nil
+}

```

Comment:
nit: `noSigner` and `noVerifier` can be combined as a struct can implement `Signer` and `Verifier` at the same time.

Reply:
Done!